core/templates/pages/subtopic-viewer-page/navbar-pre-logo-action/subtopic-viewer-navbar-pre-logo-action.component.spec.ts

```diff
@@ -15,38 +15,47 @@
 /**
  * @fileoverview Unit tests for the subtopic viewer pre logo action
  */
-import { TestBed } from '@angular/core/testing';
+
+import { ComponentFixture, TestBed, async } from '@angular/core/testing';
+import { SubtopicViewerNavbarPreLogoActionComponent } from
+  './subtopic-viewer-navbar-pre-logo-action.component';
```

Comment:
you can put import statements on the same line

Reply:
done